openstack/neutron/templates/etc/_dnsmasq.conf.tpl

```diff
@@ -6,3 +6,4 @@ no-negcache
 {{- range .Values.dnsmasq.dhcp_options }}
 dhcp-option={{ . }}
 {{- end }}
+local=/{{ .Values.dns_local_domain }}/
```

Comment:
Please use a `required` helm function for the value to ensure that local is not defined if the value is empty.

Reply:
Are you sure we want to require this or should we add an `if` around it, that only sets `local` if we have a `dns_local_domain` set to something?

Reply:
If we plan to roll it out worldwide on not specifically for some regions (I don't see a reason why have this "feature" only in selected regions), then it's fine to go with required.
Just wanted to ensure that we don't roll out this configuration without a valid value (e.g. due to missing --values or so)